Documentation/flexvolume.md

```diff
@@ -44,19 +59,42 @@ The kubelet's systemD unit file can be located at: `/etc/systemd/system/kubelet.
 
 Continue with [configuring the FlexVolume path](#configuring-the-flexvolume-path) to configure Rook to use the FlexVolume path.
 
+### Google Kubernetes Engine (GKE)
+
+Google's Kubernetes Engine uses a non-standard FlexVolume plugin directory: `/home/kubernetes/flexvolume`
+The kubelet on GKE is already configured to use that directory.
+
+Continue with [configuring the FlexVolume path](#configuring-the-flexvolume-path) to configure Rook to use the FlexVolume path.
+
 ### Kubespray
 
-Kubespray uses a non-standard [FlexVolume plugin directory](https://github.com/kubernetes-sigs/kubespray/blob/master/roles/kubernetes/node/defaults/main.yml#L55): `/var/lib/kubelet/volume-plugins`.
+#### Prior to v2.11.0
+
+Kubespray prior to the v2.11.0 release [used a non-standard FlexVolume plugin directory](https://github.com/kubernetes-sigs/kubespray/blob/f47a66622743aa31970cebeca7968a0939cb700d/roles/kubernetes/node/defaults/main.yml#L53): `/var/lib/kubelet/volume-plugins`.
 The Kubespray configured kubelet is already configured to use that directory.
 
-Continue with [configuring the FlexVolume path](#configuring-the-flexvolume-path) to configure Rook to use the FlexVolume path.
+If you are using kubespray v2.10.x or older, continue with [configuring the FlexVolume path](#configuring-the-flexvolume-path) to configure Rook to use the FlexVolume path.
+
+#### As of v2.11.0 and newer
+
+Kubespray v2.11.0 included https://github.com/kubernetes-sigs/kubespray/pull/4752 which sets the same plugin directory assumed by rook by default: `/usr/libexec/kubernetes/kubelet-plugins/volume/exec`.
+
+No special configuration of the directory is needed either in the kubelet or in rook unless you have explicitly defined a custom path.
```

Comment:
Do you think mentioning the custom flexvolume dir path variable in brackets here makes sense? That people which variable is written about here.

Reply:
Yep, let me know what you think about this one.

I did a little more research and found that kubespray actually will automatically choose /var/lib/kubelet/volumeplugins if /usr is not writable, so I included that as well.